ws-nextjs-app/cypress/support/commands.ts

```diff
@@ -17,9 +17,42 @@ declare global {
         service: Services;
         pageType: PageTypes;
         variant?: Variants;
-        id: string;
+        id?: string;
         // eslint-disable-next-line @typescript-eslint/no-explicit-any
       }) => Chainable<any>;
     }
   }
 }
+
+const getPageData = ({
+  service,
+  pageType,
+  variant,
+  id,
+}: {
+  service: Services;
+  pageType: PageTypes;
+  variant?: Variants;
+  id?: string;
+}) => {
+  const baseUrl =
+    Cypress.env('APP_ENV') === 'local'
+      ? 'http://localhost:7081'
+      : Cypress.config().baseUrl;
+  let path = `/${service}/${pageType}/${id}${variant ? `/${variant}` : ''}`;
+
+  if (service === 'ws') {
+    path = '/ws/languages';
+  }
+
+  return cy
+    .request({
+      url: `${baseUrl}${path}`,
+      failOnStatusCode: false,
+    })
+    .then(response => {
+      return response.body;
+    });
+};
+
+Cypress.Commands.add('getPageData', getPageData);
```

Comment:
Let's rename this for consistency with the other command, so that it'll be easier to migrate when the time comes 
```suggestion
Cypress.Commands.add('getPageDataFromWindow', getPageDataFromWindow);
```